src/pages/docs/projects/project-triggers/external-feed-triggers.md

```diff
@@ -0,0 +1,77 @@
+
+If you combine external feed triggers with the automatic deployment feature within [Lifecycles phases](/docs/releases/lifecycles/#Lifecycles-LifecyclePhases), whenever a package is pushed to a selected external feed, the trigger can create a release, and have it be automatically deployed.
+
+## Trigger sources
+
+Any container images or helm charts referenced in your project's deployment process can be selected to trigger release creation.  Feeds or packages referenced using variable substitution are also options here, however they will only be used by the trigger if they are both evaluated as either container/helm chart repositories, and do not use variable unavailable at release creation time.  For example, using an environment name variable will not work, because that value is only available at deploy time.
+
+:::figure
+![Package selection](/docs/projects/project-triggers/images/external-feed-trigger-packages.png)
+:::
+
+:::div{.hint}
+If you have a chain of dependencies with your external feed packages, make sure your trigger uses the package which will be pushed to its repository last.
+
+The release number that is created is guided by the Release Versioning settings under **Deployments Settings**. It will use the rules defined there.
+:::
+
+## Troubleshooting {#ExternalFeedTriggers-Troubleshooting}
+
+When you are using external feed triggers there are a few reasons why a release may not be created successfully. Take some time to consider the following troubleshooting steps:
+
+1. **Inspect the task list** for errors in the **Task** menu - Octopus will log the reason why external feed triggers failed as errors or warnings.  Note that external feed triggers are system tasks, and do not display in the list by default.  Use the **SHOW ADVANCED FILTERS** option and select **Include system tasks** to show them.
+
+2. Ensure you are pushing the package to a **supported external feed type**. While capability has been verified against most major docker providers, compatibility is not guaranteed - please contact octopus support if you encounter any problems. 
+
+3. Ensure that packages in the external feed match the [channel rules](/docs/releases/channels#version-rules) if defined for the trigger's channel (or the default channel if your project doesn't have multiple channels).  **Triggers will only create a new release if the packages match channel rules.**
+
+4. Ensure you are pushing a **new version** of the package - Octopus will not create a release where the package has already been used for creating a release.
+
+5. Ensure you are pushing a package that Octopus will consider as the **latest available package**.  The trigger's version evaluator uses SemVer, and will not triggers off image tags such as 'latest'.
+
+6. Make sure that the feed and package references **only use variables which are able to be evaluated** at release creation time.
```

Comment:
```suggestion
6. Make sure that the feed and package references only use variables which are **able to be evaluated at release creation time.**
```

Reply:
We should specify what this means, eg. "doesn't evaluate environment variables"